test/e2e/storage/testsuites/multivolume.go

```diff
@@ -316,7 +318,7 @@ func (t *multiVolumeTestSuite) DefineTests(driver storageframework.TestDriver, p
 	// [        node1        ]
 	//   |                 |     <- same volume mode
 	// [volume1]   ->  [restored volume1 snapshot]
-	ginkgo.It("should concurrently access the volume and restored snapshot from pods on the same node [LinuxOnly][Feature:VolumeSnapshotDataSource][Feature:VolumeSourceXFS]", func(ctx context.Context) {
+	f.It("should concurrently access the volume and restored snapshot from pods on the same node [LinuxOnly]", feature.VolumeSnapshotDataSource, feature.VolumeSourceXFS, func(ctx context.Context) {
```

Comment:
This shows how tests will be registered going forward:
- `f.It` is a shorthand for `framework.It`, a wrapper around `ginkgo.It` which inserts the traditional inline text tags in addition to adding Ginkgo labels.
- `feature.VolumeSnapshotDataSource` is one of the registered feature tags from https://github.com/kubernetes/kubernetes/blob/master/test/e2e/feature/feature.go. No more typos and better visibility when new features get added... ideally those then should also come with documentation of what they mean. We don't have that for existing features.